### PR TITLE
[ci-boostrap] Add second call to the mapper to ingest interface-info

### PR DESCRIPTION
**Pull request by pablintino**
This change is part of the content of https://github.com/openstack-k8s-operators/ci-framework/pull/2371, where the changes applied here are tested all along other related ones.

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/2401

### CODE REVIEW

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Review by lewisdenny — APPROVED** [review]
/approve

File: zuul.d/base.yaml [hunk 1]
```diff
@@ -305,4 +307,4 @@
     pre-run:
       - ci/playbooks/e2e-prepare.yml
       - ci/playbooks/dump_zuul_data.yml
-      - ci/playbooks/pre-ci-bootstrap.yml
+      - ci/playbooks/bootstrap-networking-mapper.yml
```
Comment (marios) [inline zuul.d/base.yaml:310]: came here via slack request, so i don't have the context. lgtm (would be nice to have more of a commit message for explanation) and i cannot find any other references to pre-ci-bootstrap than this one with code search https://github.com/search?q=org%3Aopenstack-k8s-operators%20pre-ci-bootstrap.yml&type=code

**Review by marios — APPROVED** [review]

File: zuul.d/base.yaml (hunk 1 above)
Comment (pablintino) [reply zuul.d/base.yaml:310]: @marios thanks for having a look. I've added more information in the commit message.
Yes, that playbook and the job itself, weren't used, so we can now tweak everything without worrying about breaking working jobs.

**Comment by pablintino** [comment]
recheck

File: zuul.d/base.yaml (hunk 1 above)
Comment (lewisdenny) [reply zuul.d/base.yaml:310]: @marios could you please leave a /lgtm comment :)

**Review by marios — APPROVED** [review]
/lgtm

**Pull request merged**